iGCS/Models/Craft/GCSCraftArduCopter.m

```diff
@@ -7,6 +7,12 @@
 //
 
 #import "GCSCraftArduCopter.h"
+#import "GCSCraftNotifications.h"
+
+@interface GCSCraftArduCopter ()
+@property (nonatomic, assign) mavlink_heartbeat_t lastHeartbeat;
```

Comment:
Do we need to keep lastHeartbeat as a property? Looks like this can be dropped, with a shuffle of the lines in update.